pages/features/form-analytics.vue

```diff
@@ -0,0 +1,245 @@
+              class="hero__invert__button mt-4"
+              >Get Started - it's Free</a
+            >
+          </div>
+          <div
+            class="col-lg-7 d-flex align-items-center justify-content-center mt-lg-0 mt-5"
+          >
+            <nuxt-img
+              src="/features/form-analytics/form-analytics.svg"
+              alt="Hero-Image"
+              class="img-fluid hero__image"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="container py-5">
+      <h2 class="section__heading text-center">How does Web Form Analytics help?</h2>
+      <div class="row py-5">
+        <FeatureDetail
+          :feature="feature"
+          class="col-md-6 my-3"
+          v-for="feature in features"
+          :key="feature.title"
+        />
+      </div>
+    </div>
+    <div class="container py-5">
+      <h2 class="section__heading">
+        Automate Data Collection with Formester 
+      </h2>
+      <div
+          class="d-flex flex-column flex-md-row justify-content-between mt-md-5"
+        >
+          <div
+            class="d-flex flex-column align-items-center align-items-md-start text-center text-md-start px-4 col-md-4 my-3 mt-lg-5"
+          >
+            <nuxt-img
+              style="width: 69px"
+              src="/features/form-analytics/mobile.svg"
+              alt="Formester"
+            />
+            <h4 class="sub__section-heading mt-4">Mobile Responsive</h4>
+            <p class="subheading__text">
+              Formester forms are mobile responsive and work well on a variety of devices like Tablet, Android Mobiles, Ios Mobiles and Laptop.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-column align-items-center align-items-md-start text-center text-md-start px-4 col-md-4 my-3 mt-lg-5"
+          >
+            <nuxt-img
+              style="width: 69px"
+              src="/features/form-analytics/spam-protection.svg"
+              alt="Formester"
+            />
+            <h4 class="sub__section-heading mt-4">AI Spam Protection</h4>
+            <p class="subheading__text">
+              Our AI Spam protection provides an invisible protection from Spams and Bots without using any invasive means of Captcha.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-column align-items-center align-items-md-start text-center text-md-start px-4 col-md-4 my-3 mt-lg-5"
+          >
+            <nuxt-img
+              style="width: 69px"
+              src="/features/form-analytics/drag-n-drop.svg"
+              alt="Formester"
+            />
+            <h4 class="sub__section-heading mt-4">Drag-and-Drop Form Builder</h4>
+            <p class="subheading__text">
+              Easily update your forms based on form analytic insights with our intuitive drag and drop form builder.
+            </p>
+          </div>
+        </div>
+    </div>
```

Comment:
I think we already have some component can you check that out.

Reply:
@dhaladitya108  I took this from the page you have asked to take reference from .

Reply:
but i will check this once and if possible , i will make a component if not already present